src/components/user/unavailability/unavailability.service.ts

```diff
@@ -93,18 +93,22 @@ export class UnavailabilityService {
   @HandleIdLookup(Unavailability)
   async readOne(id: ID, session: Session): Promise<Unavailability> {
     const result = await this.repo.readOne(id, session);
-    if (!result) {
-      throw new NotFoundException('Could not find user', 'user.id');
-    }
+    return await this.secure(result, id, session);
+  }
 
+  private async secure(
+    dto: UnsecuredDto<Unavailability>,
+    id: ID,
```

Comment:
Separate `id` param is not needed because `dto.id`

Reply:
Fixed